src/components/ScreenWrapper/propTypes.js

```diff
@@ -1,4 +1,5 @@
 import PropTypes from 'prop-types';
+import * as windowDimensionsPropTypes from '../withWindowDimensions';
```

Comment:
```suggestion
import {windowDimensionsPropTypes} from '../withWindowDimensions';
```

This is adding a dev/console error